docs/development/Configuration.md

```
@@ -1,33 +1,29 @@
-![Cleanflight Gui](Screenshots/cleanflight-gui.png)
+![Betaflight App](img/betaflight_configurator_welcome.png)
 
-The GUI tool is the preferred way of configuration. The GUI tool also includes a terminal which
-can be used to interact with the CLI.
+The [Betaflight App](https://app/.betaflight.com) is the preferred way of configuration. The app also includes a terminal which can be used to interact with the CLI.
 
-[Cleanflight Configurator on Chrome store](https://chrome.google.com/webstore/detail/cleanflight-configurator/enacoimjcgeinfnnnpajinjgmkahmfgb)
+If you cannot use the latest version of the app to access the FC due to firmware compatibility issues you can still access the FC via the CLI to backup your settings, or you can install an old version of the app.
 
-If you cannot use the latest version of the GUI to access the FC due to firmware compatibility issues you can still access the FC via the CLI to backup your settings, or you can install an old version of the configurator.
+Old versions of the app (formerly the configurator) can be downloaded from the releases page: https://github.com/betalfight/betaflight-configurator/releases
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)